### ISSUE TITLE
Create leaderboard

### ISSUE THREAD

**Issue by hericlesme** [hericlesme/i-just-want-to-play-tetris#6]
Create a ranking with the top scores and allows a player to register your name and score.

**Comment by c-grigsby** [comment]
Greetings! I'm thinking about implementing this and would like your thoughts. I'm considering making a small backend for the app to handle data persistence and filtering for highest scores. On the UI, top 5 scores and players could be shown on #home when the app is opened and at #game-over give the player the option to register name and score. Thanks!

**Comment by 0xHericles** [comment]
Hi, @c-grigsby! Nice to see you around. Yeah, I think you're on the right way. And for persisting data, maybe a JSON file can handle the job. Thoughts?

**Comment by c-grigsby** [comment]
Yeah, I was thinking about doing something lightweight with Express and with the current requirements a simple JSON file seems like up for the job of persisting the data. Do you think we should place the backend in a separate repo or within a directory in the current one?

**Comment by c-grigsby** [comment]
Yeah, I was thinking about doing something lightweight with Express and with the current requirements a simple JSON file seems lup for the job of persisting the data. Do you think we should place the backend in a separate repo or within a directory in the current one?

**Comment by c-grigsby** [comment]
Yeah, I was thinking about doing something lightweight with Express and with the current requirements a simple JSON file seems up for the job of persisting data. Do you think we should place the backend in a separate repo or within a directory in the current one?

**Comment by 0xHericles** [comment]
Let's put in here for now. Maybe we could even put the frontend over Next.JS and Setup API Routes, but I don't want to do any breaking changes for now, 'cause @kihuni is actually working on the webpack issue

**Comment by 0xHericles** [comment]
But since it's done, we could even use some env/workspace manager like lerna or NX, and create workflow files to deploy the apps according to changes made on the push.

**Comment by 0xHericles** [comment]
For now, just create a new folder to put the project

**Comment by c-grigsby** [comment]
Nice, yep migrating would definitely simplify some things and a few workflows would be awesome. I'll take a look into implementing this feature in the near future. Thanks for your time and have an awesome day!

**Comment by kihuni** [comment]
Hi, @monuelo  Sorry for not responding sooner. I will be working on the Webpack issue, and it will be done before the weekend's end. I hope I am not inconvenient anything. The Time zone here is East African time.

**Comment by kihuni** [comment]
Hi, @monuelo. I have already started working on the issues. Possibly by the end of tomorrow, eastern African time, I will have been done with it.

**Comment by 0xHericles** [comment]
Awesome, @kihuni! ✨ Don't worry about the time, there's no rush.  😉

**Comment by kihuni** [comment]
Thank you, I will take my time.

**Comment by c-grigsby** [comment]
Greetings! 
I recently submitted a PR for Part 2 of this issue, but as mentioned please feel free to take your time on a review as there were a number of changes involved. Also, I resolved a merge conflict to develop which was my own fault for continuing on after adjusting a few things after the back-end was deployed. I cloned a new repo to git rid of some goofy stuff I did to the old one in the process :laughing: so if you could give it a :star: sometime I'm sure it would appreciate some love. haha Hopefully it won't give any trouble now, but let me know if I need to fix anything. 

As you might know, there is an option on Heroku to automate deployment when a specific branch is pushed (which if using the root project to deploy will need to happen anyways), just let me know if you'd like me to look into it sometime. Alternatively, we could setup some workflows eventually. 

Thank you for your time and have a great day!
